server/plugin.go

```diff
@@ -26,6 +29,7 @@ const (
 	falseString = "false"
 
 	zoomProviderName = "Zoom"
+	pluginURLPath    = "/plugins/" + botUserName
```

Comment:
Can we compute the URL in a separate function like this? https://github.com/mattermost/mattermost-plugin-jira/blob/dc4266c61871200cc79d5a9feb97676aa330d50b/server/plugin.go#L396